src/pybind/mgr/dashboard/controllers/iscsi.py

```diff
@@ -64,7 +65,29 @@ def status(self):
     @Endpoint()
     @ReadPermission
     def settings(self):
-        return IscsiClient.instance().get_settings()
+        settings = IscsiClient.instance().get_settings()
+        if 'target_controls_limits' in settings:
+            target_default_controls = settings['target_default_controls']
+            for ctrl_k, ctrl_v in target_default_controls.items():
+                limits = settings['target_controls_limits'].get(ctrl_k, {})
+                if 'type' not in limits:
+                    # default
+                    limits['type'] = 'int'
+                    # backward compatibility
+                    if target_default_controls[ctrl_k] in ['Yes', 'No']:
```

Comment:
Are the strings always uppercase? Isn't it possible that they might be lowercase?

Reply:
Only 'Yes' or 'No' (uppercase) are possible here (backward compatibility).